### DISCUSSION TITLE
Tuning the v4 model hyper-params

### DISCUSSION THREAD

**Discussion by JJun-Guo** [snakers4/silero-vad#268] [Q&A] [answered]
Is the v4-model more muted? The same speech segment (including a few vocals) has output through the v3- model, but there is no output after the v4-model, speech_timestamps is an empty list.

**Comment by snakers4** [comment]
Please provide your audio files and probability graphs (`return_prob=True`).

**Comment by JJun-Guo** [comment]
> Please provide your audio files and probability graphs (`return_prob=True`).

The audio path is  https://github.com/JJ-Guo1996/AMR-code/blob/main/audio.wav

**Comment by JJun-Guo** [comment]
How about your test result?

**Comment by adamnsandle** [comment, marked as answer]
v4 was trained not to respond to background voice

As i can see, v4 finds speech in you example:
![image](https://user-images.githubusercontent.com/36505480/200785927-14b27a28-78c6-4833-933f-9853d851340b.png)

you may need to tune `min_speech_duration_ms` parameter in `get_speech_timestamps` (default value is 250ms): 

```python3
speech_timestamps = get_speech_timestamps(wav, model, sampling_rate=SAMPLING_RATE, min_speech_duration_ms=50)
print(speech_timestamps)  # [{'start': 3616, 'end': 4951}]
```

**Comment by JJun-Guo** [comment]
Why with the same parameter settings（v3 with min_speech_duration_ms=250）, v3 can get output but v4 has no output? Did v4 do any optimizations?
